{
	"alias": "FEDORA-2024-6fa5af9ea8",
	"autokarma": true,
	"autotime": true,
	"bugs": [
		{
			"bug_id": 2248505,
			"feedback": [
				{
					"bug_id": 2248505,
					"comment": {
						"id": 3361032,
						"karma": 1,
						"timestamp": "2024-01-24 13:06:53",
						"update_id": 580169,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/8364563f7a3e319a0f7ddf27dbe9945f8351d415cc7ae99cd3275291defec2b0?s=24&d=retro",
							"email": "eugenemah@gmail.com",
							"groups": [
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedora-hams"
								}
							],
							"id": 2935,
							"name": "imabug",
							"openid": "imabug.id.fedoraproject.org"
						},
						"user_id": 2935
					},
					"comment_id": 3361032,
					"karma": 1
				}
			],
			"title": "sudo-1_9_15p5 is available",
			"bugzilla": {
				"bug_id": "2248505",
				"creation_ts": "2023-11-07 12:42:11 +0000",
				"short_desc": "sudo-1_9_15p5 is available",
				"delta_ts": "2024-01-24 10:50:13 +0000",
				"bug_status": "MODIFIED",
				"keywords": "FutureFeature, Triaged",
				"priority": "unspecified",
				"bug_severity": "unspecified",
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "17744070",
						"comment_count": "0",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-11-07 12:42:11 +0000",
						"thetext": "Releases retrieved: 1_9_15\nUpstream release that is considered latest: 1_9_15\nCurrent version/release in rawhide: 1.9.14-1.p3.fc39\nURL: https://www.sudo.ws\n\nPlease consult the package updates policy before you issue an update to a stable branch: https://docs.fedoraproject.org/en-US/fesco/Updates_Policy/\n\n\nMore information about the service that created this bug can be found at: https://docs.fedoraproject.org/en-US/package-maintainers/Upstream_Release_Monitoring\n\n\nPlease keep in mind that with any upstream change, there may also be packaging changes that need to be made. Specifically, please remember that it is your responsibility to review the new version to ensure that the licensing is still correct and that no non-free or legally problematic items have been added upstream.\n\n\nBased on the information from Anitya: https://release-monitoring.org/project/4906/\n\n\nTo change the monitoring settings for the project, please visit:\nhttps://src.fedoraproject.org/rpms/sudo"
					},
					{
						"isprivate": "0",
						"commentid": "17744071",
						"comment_count": "1",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-11-07 12:42:17 +0000",
						"thetext": "Scratch build failed. Details below:\n\nBuilderException: Build failed:\nCommand '['rpmbuild', '-D', '_sourcedir .', '-D', '_topdir .', '-bs', '/var/tmp/thn-rcva7a72/sudo.spec']' returned non-zero exit status 1.\n\nStdOut:\nsetting SOURCE_DATE_EPOCH=1699315200\nerror: Bad file: ./sudo-1_9_15p3.tar.gz: No such file or directory\n\nRPM build errors:\n    Bad file: ./sudo-1_9_15p3.tar.gz: No such file or directory\n\n\nTraceback:\n  File \"/usr/local/lib/python3.11/site-packages/hotness/use_cases/package_scratch_build_use_case.py\", line 56, in build\n    result = self.builder.build(request.package, request.opts)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.11/site-packages/hotness/builders/koji.py\", line 229, in build\n    raise BuilderException(\n\nIf you think this issue is caused by some bug in the-new-hotness, please report it on the-new-hotness issue tracker: https://github.com/fedora-infra/the-new-hotness/issues"
					},
					{
						"isprivate": "0",
						"commentid": "17744072",
						"comment_count": "2",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-11-07 12:42:18 +0000",
						"thetext": "Created attachment 1997607\nUpdate to 1_9_15 (#2248505)"
					},
					{
						"isprivate": "0",
						"commentid": "17751239",
						"comment_count": "3",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-11-10 20:00:50 +0000",
						"thetext": "Releases retrieved: 1_9_15p1, 1_9_15p2\nUpstream release that is considered latest: 1_9_15p2\nCurrent version/release in rawhide: 1.9.14-1.p3.fc39\nURL: https://www.sudo.ws\n\nPlease consult the package updates policy before you issue an update to a stable branch: https://docs.fedoraproject.org/en-US/fesco/Updates_Policy/\n\n\nMore information about the service that created this bug can be found at: https://docs.fedoraproject.org/en-US/package-maintainers/Upstream_Release_Monitoring\n\n\nPlease keep in mind that with any upstream change, there may also be packaging changes that need to be made. Specifically, please remember that it is your responsibility to review the new version to ensure that the licensing is still correct and that no non-free or legally problematic items have been added upstream.\n\n\nBased on the information from Anitya: https://release-monitoring.org/project/4906/\n\n\nTo change the monitoring settings for the project, please visit:\nhttps://src.fedoraproject.org/rpms/sudo"
					},
					{
						"isprivate": "0",
						"commentid": "17751240",
						"comment_count": "4",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-11-10 20:00:54 +0000",
						"thetext": "Scratch build failed. Details below:\n\nBuilderException: Build failed:\nCommand '['rpmbuild', '-D', '_sourcedir .', '-D', '_topdir .', '-bs', '/var/tmp/thn-fdmypl_u/sudo.spec']' returned non-zero exit status 1.\n\nStdOut:\nsetting SOURCE_DATE_EPOCH=1699574400\nerror: Bad file: ./sudo-1_9_15p2p3.tar.gz: No such file or directory\n\nRPM build errors:\n    Bad file: ./sudo-1_9_15p2p3.tar.gz: No such file or directory\n\n\nTraceback:\n  File \"/usr/local/lib/python3.11/site-packages/hotness/use_cases/package_scratch_build_use_case.py\", line 56, in build\n    result = self.builder.build(request.package, request.opts)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.11/site-packages/hotness/builders/koji.py\", line 229, in build\n    raise BuilderException(\n\nIf you think this issue is caused by some bug in the-new-hotness, please report it on the-new-hotness issue tracker: https://github.com/fedora-infra/the-new-hotness/issues"
					},
					{
						"isprivate": "0",
						"commentid": "17751241",
						"comment_count": "5",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-11-10 20:00:55 +0000",
						"thetext": "Created attachment 1998475\nUpdate to 1_9_15p2 (#2248505)"
					},
					{
						"isprivate": "0",
						"commentid": "17806472",
						"comment_count": "6",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-12-14 13:52:28 +0000",
						"thetext": "Releases retrieved: 1_9_15p3\nUpstream release that is considered latest: 1_9_15p3\nCurrent version/release in rawhide: 1.9.14-1.p3.fc39\nURL: https://www.sudo.ws\n\nPlease consult the package updates policy before you issue an update to a stable branch: https://docs.fedoraproject.org/en-US/fesco/Updates_Policy/\n\n\nMore information about the service that created this bug can be found at: https://docs.fedoraproject.org/en-US/package-maintainers/Upstream_Release_Monitoring\n\n\nPlease keep in mind that with any upstream change, there may also be packaging changes that need to be made. Specifically, please remember that it is your responsibility to review the new version to ensure that the licensing is still correct and that no non-free or legally problematic items have been added upstream.\n\n\nBased on the information from Anitya: https://release-monitoring.org/project/4906/\n\n\nTo change the monitoring settings for the project, please visit:\nhttps://src.fedoraproject.org/rpms/sudo"
					},
					{
						"isprivate": "0",
						"commentid": "17806473",
						"comment_count": "7",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-12-14 13:52:33 +0000",
						"thetext": "Scratch build failed. Details below:\n\nBuilderException: Build failed:\nCommand '['rpmbuild', '-D', '_sourcedir .', '-D', '_topdir .', '-bs', '/var/tmp/thn-hav11dqy/sudo.spec']' returned non-zero exit status 1.\n\nStdOut:\nsetting SOURCE_DATE_EPOCH=1702512000\nerror: Bad file: ./sudo-1_9_15p3p3.tar.gz: No such file or directory\n\nRPM build errors:\n    Bad file: ./sudo-1_9_15p3p3.tar.gz: No such file or directory\n\n\nTraceback:\n  File \"/usr/local/lib/python3.11/site-packages/hotness/use_cases/package_scratch_build_use_case.py\", line 56, in build\n    result = self.builder.build(request.package, request.opts)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.11/site-packages/hotness/builders/koji.py\", line 229, in build\n    raise BuilderException(\n\nIf you think this issue is caused by some bug in the-new-hotness, please report it on the-new-hotness issue tracker: https://github.com/fedora-infra/the-new-hotness/issues"
					},
					{
						"isprivate": "0",
						"commentid": "17806474",
						"comment_count": "8",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-12-14 13:52:34 +0000",
						"thetext": "Created attachment 2004296\nUpdate to 1_9_15p3 (#2248505)"
					},
					{
						"isprivate": "0",
						"commentid": "17808735",
						"comment_count": "9",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-12-16 10:00:24 +0000",
						"thetext": "Releases retrieved: 1_9_15p4\nUpstream release that is considered latest: 1_9_15p4\nCurrent version/release in rawhide: 1.9.14-1.p3.fc39\nURL: https://www.sudo.ws\n\nPlease consult the package updates policy before you issue an update to a stable branch: https://docs.fedoraproject.org/en-US/fesco/Updates_Policy/\n\n\nMore information about the service that created this bug can be found at: https://docs.fedoraproject.org/en-US/package-maintainers/Upstream_Release_Monitoring\n\n\nPlease keep in mind that with any upstream change, there may also be packaging changes that need to be made. Specifically, please remember that it is your responsibility to review the new version to ensure that the licensing is still correct and that no non-free or legally problematic items have been added upstream.\n\n\nBased on the information from Anitya: https://release-monitoring.org/project/4906/\n\n\nTo change the monitoring settings for the project, please visit:\nhttps://src.fedoraproject.org/rpms/sudo"
					},
					{
						"isprivate": "0",
						"commentid": "17808736",
						"comment_count": "10",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-12-16 10:00:29 +0000",
						"thetext": "Scratch build failed. Details below:\n\nBuilderException: Build failed:\nCommand '['rpmbuild', '-D', '_sourcedir .', '-D', '_topdir .', '-bs', '/var/tmp/thn-2krjei6z/sudo.spec']' returned non-zero exit status 1.\n\nStdOut:\nsetting SOURCE_DATE_EPOCH=1702684800\nerror: Bad file: ./sudo-1_9_15p4p3.tar.gz: No such file or directory\n\nRPM build errors:\n    Bad file: ./sudo-1_9_15p4p3.tar.gz: No such file or directory\n\n\nTraceback:\n  File \"/usr/local/lib/python3.11/site-packages/hotness/use_cases/package_scratch_build_use_case.py\", line 56, in build\n    result = self.builder.build(request.package, request.opts)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.11/site-packages/hotness/builders/koji.py\", line 229, in build\n    raise BuilderException(\n\nIf you think this issue is caused by some bug in the-new-hotness, please report it on the-new-hotness issue tracker: https://github.com/fedora-infra/the-new-hotness/issues"
					},
					{
						"isprivate": "0",
						"commentid": "17808737",
						"comment_count": "11",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-12-16 10:00:30 +0000",
						"thetext": "Created attachment 2004563\nUpdate to 1_9_15p4 (#2248505)"
					},
					{
						"isprivate": "0",
						"commentid": "17819896",
						"comment_count": "12",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-12-31 23:18:58 +0000",
						"thetext": "Releases retrieved: 1_9_15p5\nUpstream release that is considered latest: 1_9_15p5\nCurrent version/release in rawhide: 1.9.14-1.p3.fc39\nURL: https://www.sudo.ws\n\nPlease consult the package updates policy before you issue an update to a stable branch: https://docs.fedoraproject.org/en-US/fesco/Updates_Policy/\n\n\nMore information about the service that created this bug can be found at: https://docs.fedoraproject.org/en-US/package-maintainers/Upstream_Release_Monitoring\n\n\nPlease keep in mind that with any upstream change, there may also be packaging changes that need to be made. Specifically, please remember that it is your responsibility to review the new version to ensure that the licensing is still correct and that no non-free or legally problematic items have been added upstream.\n\n\nBased on the information from Anitya: https://release-monitoring.org/project/4906/\n\n\nTo change the monitoring settings for the project, please visit:\nhttps://src.fedoraproject.org/rpms/sudo"
					},
					{
						"isprivate": "0",
						"commentid": "17819897",
						"comment_count": "13",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-12-31 23:19:01 +0000",
						"thetext": "Scratch build failed. Details below:\n\nBuilderException: Build failed:\nCommand '['rpmbuild', '-D', '_sourcedir .', '-D', '_topdir .', '-bs', '/var/tmp/thn-nxikhuh0/sudo.spec']' returned non-zero exit status 1.\n\nStdOut:\nsetting SOURCE_DATE_EPOCH=1703980800\nerror: Bad file: ./sudo-1_9_15p5p3.tar.gz: No such file or directory\n\nRPM build errors:\n    Bad file: ./sudo-1_9_15p5p3.tar.gz: No such file or directory\n\n\nTraceback:\n  File \"/usr/local/lib/python3.11/site-packages/hotness/use_cases/package_scratch_build_use_case.py\", line 56, in build\n    result = self.builder.build(request.package, request.opts)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.11/site-packages/hotness/builders/koji.py\", line 229, in build\n    raise BuilderException(\n\nIf you think this issue is caused by some bug in the-new-hotness, please report it on the-new-hotness issue tracker: https://github.com/fedora-infra/the-new-hotness/issues"
					},
					{
						"isprivate": "0",
						"commentid": "17819898",
						"comment_count": "14",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2023-12-31 23:19:03 +0000",
						"thetext": "Created attachment 2006704\nUpdate to 1_9_15p5 (#2248505)"
					},
					{
						"isprivate": "0",
						"commentid": "17855699",
						"comment_count": "15",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-01-24 10:21:07 +0000",
						"thetext": "FEDORA-2024-57eebd0963 has been submitted as an update to Fedora 40. https://bodhi.fedoraproject.org/updates/FEDORA-2024-57eebd0963"
					},
					{
						"isprivate": "0",
						"commentid": "17855728",
						"comment_count": "16",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-01-24 10:48:47 +0000",
						"thetext": "FEDORA-2024-cdccda4f62 has been submitted as an update to Fedora 39. https://bodhi.fedoraproject.org/updates/FEDORA-2024-cdccda4f62"
					},
					{
						"isprivate": "0",
						"commentid": "17855730",
						"comment_count": "17",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-01-24 10:50:13 +0000",
						"thetext": "FEDORA-2024-6fa5af9ea8 has been submitted as an update to Fedora 38. https://bodhi.fedoraproject.org/updates/FEDORA-2024-6fa5af9ea8"
					}
				]
			}
		},
		{
			"bug_id": 2255569,
			"feedback": [
				{
					"bug_id": 2255569,
					"comment": {
						"id": 3361032,
						"karma": 1,
						"timestamp": "2024-01-24 13:06:53",
						"update_id": 580169,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/8364563f7a3e319a0f7ddf27dbe9945f8351d415cc7ae99cd3275291defec2b0?s=24&d=retro",
							"email": "eugenemah@gmail.com",
							"groups": [
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedora-hams"
								}
							],
							"id": 2935,
							"name": "imabug",
							"openid": "imabug.id.fedoraproject.org"
						},
						"user_id": 2935
					},
					"comment_id": 3361032
				}
			],
			"security": true,
			"title": "TRIAGE CVE-2023-42465 sudo: Targeted Corruption of Register and Stack Variables [fedora-all]",
			"bugzilla": {
				"bug_id": "2255569",
				"creation_ts": "2023-12-21 23:04:21 +0000",
				"short_desc": "TRIAGE CVE-2023-42465 sudo: Targeted Corruption of Register and Stack Variables [fedora-all]",
				"delta_ts": "2024-01-24 10:50:16 +0000",
				"bug_status": "MODIFIED",
				"keywords": "Security, SecurityTracking",
				"priority": "medium",
				"bug_severity": "medium",
				"blocked": [
					{
						"bug_id": "2255568",
						"alias": "mayhem",
						"creation_ts": "2023-12-21 23:04:14 +0000",
						"short_desc": "CVE-2023-42465 sudo: Targeted Corruption of Register and Stack Variables",
						"delta_ts": "2024-01-17 14:28:51 +0000",
						"bug_status": "NEW",
						"keywords": "Security",
						"priority": "medium",
						"bug_severity": "medium",
						"depends_on": [
							"2255569"
						],
						"blocked": [
							{
								"bug_id": "2255571",
								"error": "NotPermitted"
							}
						],
						"long_desc": [
							{
								"isprivate": "0",
								"commentid": "17814267",
								"comment_count": "0",
								"who": {
									"text": "ntait",
									"name": "Nick Tait"
								},
								"bug_when": "2023-12-21 23:04:14 +0000",
								"thetext": "From https://www.openwall.com/lists/oss-security/2023/12/21/9 a new CVE has been reserved against the sudo package.\n\n'''\nOur recent paper<https://arxiv.org/pdf/2309.02545.pdf> [AsiaCCS'24] describes a potential vulnerability where stack/register variables can be flipped via fault injection, affecting execution flow in security-sensitive code. There are mitigation strategies you may be interested in incorporating into your code:\n\n Take this vulnerable code, for example:\n\nint auth = 0;\n\n//password check code that sets auth variable\n\nif(auth != 0)\n\nreturn AUTH_SUCCESS;\n\nelse\n\nreturn AUTH_FAILURE;\n\nThe idea is that any bit can be flipped in auth, and it will result in a mis-authentication. We prove this is a potential vulnerability in OpenSSH, OpenSSL, MySQL, and SUDO. To mitigate this, it is important to have tight logic such that a single-bit flip will not result in unintended execution. For example:\n\nint auth = 0xbe405d1a;\n\n// password check code that sets auth variable to 0x23ab9701 is successful\n\nIf(auth == 0x23ab9701)\n\n               return AUTH_SUCCESS;\n\nelse\n\n               return AUTH_FAILURE;\n\nIn this case, the auth variable must be corrupted into the exact authentication pattern, which is fairly improbable.\n\n\n\nWe issued CVE-2023-42465 for SUDO for this vulnerability.\n\nHere is the patch implemented in v1.9.15.\n\nhttps://github.com/sudo-project/sudo/commit/7873f8334c8d31031f8cfa83bd97ac6029309e4f\n\nPaper link: https://arxiv.org/abs/2309.02545\n\n\n\nCaner Tol\n___________________________\nWorcester Polytechnic Institute\nhttps://vernamlab.org<https://vernamlab.org/>\n'''"
							},
							{
								"isprivate": "0",
								"commentid": "17814271",
								"comment_count": "1",
								"who": {
									"text": "ntait",
									"name": "Nick Tait"
								},
								"bug_when": "2023-12-21 23:04:29 +0000",
								"thetext": "Created sudo tracking bugs for this issue:\n\nAffects: fedora-all [bug 2255569]"
							}
						]
					}
				],
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "17814269",
						"comment_count": "0",
						"who": {
							"text": "ntait",
							"name": "Nick Tait"
						},
						"bug_when": "2023-12-21 23:04:21 +0000",
						"thetext": "\nMore information about this security flaw is available in the following bug:\n\nhttp://bugzilla.redhat.com/show_bug.cgi?id=2255568\n\nDisclaimer: Community trackers are created by Red Hat Product Security team on a best effort basis. Package maintainers are required to ascertain if the flaw indeed affects their package, before starting the update process."
					},
					{
						"isprivate": "0",
						"commentid": "17814270",
						"comment_count": "1",
						"who": {
							"text": "ntait",
							"name": "Nick Tait"
						},
						"bug_when": "2023-12-21 23:04:24 +0000",
						"thetext": "Use the following template to for the 'fedpkg update' request to submit an\nupdate for this issue as it contains the top-level parent bug(s) as well as\nthis tracking bug.  This will ensure that all associated bugs get updated\nwhen new packages are pushed to stable.\n\n=====\n\n# bugfix, security, enhancement, newpackage (required)\ntype=security\n\n# low, medium, high, urgent (required)\nseverity=medium\n\n# testing, stable\nrequest=testing\n\n# Bug numbers: 1234,9876\nbugs=2255568,2255569\n\n# Description of your update\nnotes=Security fix for [PUT CVEs HERE]\n\n# Enable request automation based on the stable/unstable karma thresholds\nautokarma=True\nstable_karma=3\nunstable_karma=-3\n\n# Automatically close bugs when this marked as stable\nclose_bugs=True\n\n# Suggest that users restart after update\nsuggest_reboot=False\n\n======\n\nAdditionally, you may opt to use the bodhi web interface to submit updates:\n\nhttps://bodhi.fedoraproject.org/updates/new"
					},
					{
						"isprivate": "0",
						"commentid": "17855700",
						"comment_count": "2",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-01-24 10:21:10 +0000",
						"thetext": "FEDORA-2024-57eebd0963 has been submitted as an update to Fedora 40. https://bodhi.fedoraproject.org/updates/FEDORA-2024-57eebd0963"
					},
					{
						"isprivate": "0",
						"commentid": "17855729",
						"comment_count": "3",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-01-24 10:48:50 +0000",
						"thetext": "FEDORA-2024-cdccda4f62 has been submitted as an update to Fedora 39. https://bodhi.fedoraproject.org/updates/FEDORA-2024-cdccda4f62"
					},
					{
						"isprivate": "0",
						"commentid": "17855731",
						"comment_count": "4",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-01-24 10:50:16 +0000",
						"thetext": "FEDORA-2024-6fa5af9ea8 has been submitted as an update to Fedora 38. https://bodhi.fedoraproject.org/updates/FEDORA-2024-6fa5af9ea8"
					}
				]
			}
		}
	],
	"builds": [
		{
			"epoch": 0,
			"nvr": "sudo-1.9.15-1.p5.fc38",
			"release_id": 66,
			"signed": true,
			"type": "rpm",
			"package": {
				"aarch64": [
					{
						"name": "sudo-logsrvd-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "aarch64"
					},
					{
						"name": "sudo-devel",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "aarch64"
					},
					{
						"name": "sudo-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "aarch64"
					},
					{
						"name": "sudo-python-plugin-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "aarch64"
					},
					{
						"name": "sudo-logsrvd",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "aarch64"
					},
					{
						"name": "sudo-python-plugin",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "aarch64"
					},
					{
						"name": "sudo-debugsource",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "aarch64"
					},
					{
						"name": "sudo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "aarch64"
					}
				],
				"i686": [
					{
						"name": "sudo-debugsource",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "i686"
					},
					{
						"name": "sudo-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "i686"
					},
					{
						"name": "sudo-logsrvd-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "i686"
					},
					{
						"name": "sudo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "i686"
					},
					{
						"name": "sudo-logsrvd",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "i686"
					},
					{
						"name": "sudo-python-plugin-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "i686"
					},
					{
						"name": "sudo-python-plugin",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "i686"
					},
					{
						"name": "sudo-devel",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "i686"
					}
				],
				"ppc64le": [
					{
						"name": "sudo-logsrvd",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "ppc64le"
					},
					{
						"name": "sudo-devel",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "ppc64le"
					},
					{
						"name": "sudo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "ppc64le"
					},
					{
						"name": "sudo-logsrvd-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "ppc64le"
					},
					{
						"name": "sudo-python-plugin",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "ppc64le"
					},
					{
						"name": "sudo-python-plugin-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "ppc64le"
					},
					{
						"name": "sudo-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "ppc64le"
					},
					{
						"name": "sudo-debugsource",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "ppc64le"
					}
				],
				"s390x": [
					{
						"name": "sudo-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "s390x"
					},
					{
						"name": "sudo-debugsource",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "s390x"
					},
					{
						"name": "sudo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "s390x"
					},
					{
						"name": "sudo-logsrvd-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "s390x"
					},
					{
						"name": "sudo-logsrvd",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "s390x"
					},
					{
						"name": "sudo-python-plugin-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "s390x"
					},
					{
						"name": "sudo-python-plugin",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "s390x"
					},
					{
						"name": "sudo-devel",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "s390x"
					}
				],
				"src": [
					{
						"name": "sudo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "src"
					}
				],
				"x86_64": [
					{
						"name": "sudo-python-plugin-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "x86_64"
					},
					{
						"name": "sudo-logsrvd-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "x86_64"
					},
					{
						"name": "sudo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "x86_64"
					},
					{
						"name": "sudo-debuginfo",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "x86_64"
					},
					{
						"name": "sudo-logsrvd",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "x86_64"
					},
					{
						"name": "sudo-debugsource",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "x86_64"
					},
					{
						"name": "sudo-python-plugin",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "x86_64"
					},
					{
						"name": "sudo-devel",
						"version": "1.9.15",
						"release": "1.p5.fc38",
						"arch": "x86_64"
					}
				]
			}
		}
	],
	"close_bugs": true,
	"comments": [
		{
			"id": 3360878,
			"text": "This update has been submitted for testing by rsroka. ",
			"timestamp": "2024-01-24 10:50:09",
			"update_id": 580169,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3360879,
			"text": "This update's test gating status has been changed to 'waiting'.",
			"timestamp": "2024-01-24 10:50:11",
			"update_id": 580169,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3361011,
			"text": "This update's test gating status has been changed to 'failed'.",
			"timestamp": "2024-01-24 12:45:40",
			"update_id": 580169,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"bug_feedback": [
				{
					"bug": {
						"bug_id": 2248505,
						"title": "sudo-1_9_15p5 is available"
					},
					"bug_id": 2248505,
					"comment_id": 3361032,
					"karma": 1
				},
				{
					"bug": {
						"bug_id": 2255569,
						"security": true,
						"title": "TRIAGE CVE-2023-42465 sudo: Targeted Corruption of Register and Stack Variables [fedora-all]"
					},
					"bug_id": 2255569,
					"comment_id": 3361032
				}
			],
			"id": 3361032,
			"karma": 1,
			"timestamp": "2024-01-24 13:06:53",
			"update_id": 580169,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/8364563f7a3e319a0f7ddf27dbe9945f8351d415cc7ae99cd3275291defec2b0?s=24&d=retro",
				"email": "eugenemah@gmail.com",
				"groups": [
					{
						"name": "ipausers"
					},
					{
						"name": "fedora-contributor"
					},
					{
						"name": "signed_fpca"
					},
					{
						"name": "fedora-hams"
					}
				],
				"id": 2935,
				"name": "imabug",
				"openid": "imabug.id.fedoraproject.org"
			},
			"user_id": 2935
		}
	],
	"content_type": "rpm",
	"critpath": true,
	"critpath_groups": "core critical-path-lxqt critical-path-server",
	"date_submitted": "2024-01-24 10:50:09",
	"karma": 1,
	"notes": "Rabase to 1.9.15p5",
	"release": {
		"branch": "f38",
		"candidate_tag": "f38-updates-candidate",
		"composed_by_bodhi": true,
		"create_automatic_updates": false,
		"dist_tag": "f38",
		"eol": "2024-05-14",
		"id_prefix": "FEDORA",
		"long_name": "Fedora 38",
		"mail_template": "fedora_errata_template",
		"name": "F38",
		"override_tag": "f38-override",
		"package_manager": "dnf",
		"pending_signing_tag": "f38-signing-pending",
		"pending_stable_tag": "f38-updates-pending",
		"pending_testing_tag": "f38-updates-testing-pending",
		"stable_tag": "f38-updates",
		"state": "current",
		"testing_repository": "updates-testing",
		"testing_tag": "f38-updates-testing",
		"version": "38"
	},
	"request": "testing",
	"require_bugs": true,
	"require_testcases": true,
	"requirements": "",
	"severity": "medium",
	"stable_days": 14,
	"stable_karma": 3,
	"status": "pending",
	"suggest": "unspecified",
	"test_gating_status": "failed",
	"title": "sudo-1.9.15-1.p5.fc38",
	"type": "security",
	"url": "https://bodhi.fedoraproject.org/updates/FEDORA-2024-6fa5af9ea8",
	"unstable_karma": -3,
	"updateid": "FEDORA-2024-6fa5af9ea8",
	"user": {
		"avatar": "https://seccdn.libravatar.org/avatar/39df0f0f67488aa1503694ae2be6d7dad131b28a334a62c33418cb2cb213a81f?s=24&d=retro",
		"email": "rsroka@redhat.com",
		"groups": [
			{
				"name": "packager"
			},
			{
				"name": "ipausers"
			},
			{
				"name": "fedora-contributor"
			},
			{
				"name": "signed_fpca"
			},
			{
				"name": "fedorabugs"
			}
		],
		"id": 3185,
		"name": "rsroka",
		"openid": "rsroka.id.fedoraproject.org"
	},
	"version_hash": "8fca2f6ea2ea11783fa6057e012225341e57e8ec"
}
